instances/treasury-devdao.near/widget/lib/common.jsx

```diff
@@ -1,4 +1,5 @@
-const treasuryDaoID = "${REPL_TREASURY}";
+const treasuryDaoID = "build.sputnik-dao.near";
```

Comment:
Is this supposed to be hardcoded?

Reply:
yeah committed it by mistake, removed it